clojure-ts-mode.el

```diff
@@ -520,6 +520,10 @@ with the markdown_inline grammar."
   "Return non-nil if NODE is a Clojure keyword."
   (string-equal "kwd_lit" (treesit-node-type node)))
 
+(defun clojure-ts--quoted-var-node-p (node)
```

Comment:
I think the right terminology here is `var-quoted-node`. That's because you can also have the standard quoting with `'`.